release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -429,6 +429,30 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 
 In {product-title} {product-version}, the installation program uses the Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
 
+[id="ocp-4-19-installation-and-update-support-azure-instance-types_{context}"]
+==== Support added for additional {azure-first} instance types
+
+Additional {azure-first} instance types for machine types based on 64-bit x86 architecture have been tested with {product-title} {product-version}.
+
+For the  Dxv6 machine series, the following instances types have been tested:
+
+* `StandardDdsv6Family`
+* `StandardDldsv6Family`
+* `StandardDlsv6Family`
+* `StandardDsv6Family`
+
+For the Lsv4 and Lasv4 machine series, the following instances types have been tested:
+
+* `standardLasv4Family`
+* `standardLsv4Family`
+
+For the ND and NV machine series, the following instances types have been tested:
+
+* `StandardNVadsV710v5Family`
+* `Standard NDASv4_A100 Family`
```

Comment:
it's really weird for this to have spaces in it, but i see this is what is in QE's PR so :shrug: